src/main/kotlin/com/terraformation/backend/customer/api/OrganizationsController.kt

```diff
@@ -252,16 +256,29 @@ data class CreateOrganizationRequestPayload(
         maxLength = 6)
     val countrySubdivisionCode: String?,
     val description: String?,
+    val managedLocationTypes: List<ManagedLocationType>?,
```

Comment:
Minor nit: Could make this a `Set` instead of a `List` so the OpenAPI schema will indicate that there shouldn't be duplicate values.